tapdance/utils.go

```diff
@@ -267,7 +268,7 @@ func obfuscateTagAndProtobuf(stegoPayload []byte, protobuf []byte, stationPubkey
 	if err != nil {
 		return nil, nil, err
 	}
-	representative[31] |= (0x80 & randByte[0])
+	representative[31] |= (0x40 & randByte[0])
```

Comment:
You want 0xc0 here, not 0x40.

0x40 would still be only one bit (0b0100_0000), while 0xc0 is the upper two-bits (0b1100_0000)